### PR TITLE
a bit of cleanup around online groups

### PR DESCRIPTION
**Pull request by s-cheng**
A few small updates around online groups:

1. Only show up to 5 on the main Community page
2. Order them alphabetically instead of by date created
2. Fix the top margin on the Group Page on mobile

### CODE REVIEW

File: packages/lesswrong/components/localGroups/LocalGroupPage.tsx [hunk 1]
```diff
@@ -79,13 +84,13 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
 
   return (
     <div className={classes.root}>
-      {group.googleLocation && <CommunityMapWrapper
+      {group.googleLocation ? <CommunityMapWrapper
         terms={{view: "events", groupId: groupId}}
         groupQueryTerms={{view: "single", groupId: groupId}}
         mapOptions={{zoom:11, center: group.googleLocation.geometry.location, initialOpenWindows:[groupId]}}
-      />}
+      /> : <div className={classes.bannerImage}></div>}
```
Comment (s-cheng) [inline packages/lesswrong/components/localGroups/LocalGroupPage.tsx:91]: I'm working on adding a banner image to this page anyway (like the one for sequences), so I'm going to add this placeholder here to fix the margin issues I'm seeing.

File: packages/lesswrong/lib/generated/fragmentTypes.d.ts [hunk 2]
```diff
@@ -2007,3 +2007,4 @@ interface CollectionNamesByFragmentName {
 }
 
 type CollectionNameString = "Bans"|"Books"|"Chapters"|"Collections"|"Comments"|"Conversations"|"DatabaseMetadata"|"DebouncerEvents"|"EmailTokens"|"FeaturedResources"|"GardenCodes"|"LWEvents"|"LegacyData"|"Localgroups"|"Messages"|"Migrations"|"Notifications"|"PetrovDayLaunchs"|"PostRelations"|"Posts"|"RSSFeeds"|"ReadStatuses"|"Reports"|"ReviewVotes"|"Revisions"|"Sequences"|"Subscriptions"|"TagFlags"|"TagRels"|"Tags"|"Users"|"Votes"
+
```
Comment (s-cheng) [inline packages/lesswrong/lib/generated/fragmentTypes.d.ts:2010]: Not sure why this got added when I ran `generate`

File: packages/lesswrong/components/localGroups/LocalGroupPage.tsx [hunk 3]
```diff
@@ -14,6 +14,11 @@ import { forumTypeSetting } from '../../lib/instanceSettings';
 
 const styles = createStyles((theme: ThemeType): JssStyles => ({
   root: {},
+  bannerImage: {
+    [theme.breakpoints.up('md')]: {
+      marginTop: -50
```
Comment (jcmustin) [inline packages/lesswrong/components/localGroups/LocalGroupPage.tsx:19]: 💯

**Review by jcmustin — APPROVED** [review]
👍 !

**Pull request merged**